provdebug/pvdebug.py

```diff
@@ -107,7 +107,7 @@ def run():
                     debugTrace.current_record().prettyPrint(frame_length)
                 continue
             elif userFlag == "b" or userFlag == "back":
-                record = debugTrace.next_record()
+                record = debugTrace.prev_record()
```

Comment:
I'm a monkey

Reply:
Is that why pressing back sometimes didn't actually show the previous program? :o

Reply:
Good catch anyhow, lol